test/dlopen/test_dlopen.c

```diff
@@ -50,12 +58,8 @@ int main(int argc, char *argv[])
    int result;
 
    result = gotcha_wrap(funcs, 2, "dlopen_test");
-   if (result != 0) {
```

Comment:
Because, one of those functions isn't found, and shouldn't be (I believe, let me retest this). This is changed in another update I'm working on (the one with Wall Wextra weirdness).

Version there (will be merged soon into this one)

```
if(result != GOTCHA_FUNCTION_NOT_FOUND){
     fprintf(stderr, "GOTCHA should have failed to find a function, but found it\n");
     return -1;
   }
```